packages/manager/modules/pci/src/projects/project/ai/apps/components/update-image/update-image.html

```diff
@@ -0,0 +1,27 @@
+<oui-modal
+    data-heading="{{:: 'pci_app_update_app_image_title' | translate }}"
+    data-primary-action="$ctrl.updateAppImage()"
+    data-primary-label="{{:: 'pci_app_update_app_image_submit_label' | translate }}"
+    data-secondary-action="$ctrl.cancel()"
+    data-secondary-label="{{:: 'pci_app_update_app_image_cancel_label' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+    data-loading="$ctrl.isUpdating"
```

Comment:
Maybe you should block primary action with `primary-disabled` attribute, to avoid validating empty input.
(See: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-modal-webcomponent--simple)

Reply:
Good idea, added here: https://github.com/ovh/manager/pull/8002/commits/76e75e6987339371746a9ba8184fd89a532165da